README.md

```diff
@@ -0,0 +1,60 @@
+# iOS enhanced media player
+
+Repo for application in iOS for media player
```

Comment:
Add a brief description for this player, there **is** a application/demo for the player in this repo, but the main purpose for this repo is to be a package where we can import into projects
Add some info like: implemented in SwiftUI using native APIs (`AVFoundation`, etc)

Reply:
`EnhancedMediaPlayers` is the package name, not a API provided by Apple